internal/auth/auth.go

```diff
@@ -91,6 +73,15 @@ func CreateAuthMiddleware(cfg *config.Config, log logrus.FieldLogger) (func(http
 			if err != nil {
 				return nil, fmt.Errorf("failed to create JWT AuthN: %w", err)
 			}
+		} else if cfg.Auth.AAPGatewayApiUrl != "" {
+			aapUrl := strings.TrimSuffix(cfg.Auth.AAPGatewayApiUrl, "/")
+			log.Println(fmt.Sprintf("AAP Gateway auth enabled: %s", aapUrl))
+			authZ = NilAuth{}
```

Comment:
Could we please log at start-up which authN and authZ has been configured and - in case of no authN / authZ log a warning?